### ISSUE TITLE
Add front-end page to edit existing schema

### ISSUE THREAD

**Issue by fergcb** [team-genny/genny#13]
Depends on #11

There should be a front end page that builds on/uses SchemaForm to allow users to edit existing schemas.

**Comment by Tizzer1331** [comment]
Do we want to be taken back to main schema Page after editing?
Maybe use a toast to show the edit. this would improve UI approach for the user.

This would also solve the issue where if I edit, save and then attempt to edit again I have no indication if the edit was successful.

no CSS imported like Create Schema, maybe use a universal schema edit/create that unifies the both when one is changed affects both.

**Comment by nmcc1212** [comment]
1. No
2. we don't have a component for this and imo creating one is vastly out of scope, and the existing approach mirrors the created schema message
3. have a feature ready to go that makes the message disappear after 3 seconds 
4. there is no need to do this, it gets css from the page component, and there is no in-line css specific to the edit page that would go in the css file anyway

**Comment by Tizzer1331** [comment]
Why would you need to stay on the page after you've done your edit? 
If I've done my edit why would I want to stay on the SAME schema?

I don't entirely agree with 2. but fine, may be worth opening another issue for it to unify them into a better approach than that.

**Comment by nmcc1212** [comment]
1. it mimicks the behaviour of create schema
2. to make more edits
